trace/trace_test.go

```diff
@@ -33,7 +33,7 @@ func TestCreateTraceIfEmpty(t *testing.T) {
 				ProbabilitySample: &ps,
 			},
 			defaultProbabilitySample:  0.5,
-			expectedProbabilitySample: 0.5,
+			expectedProbabilitySample: ps,
```

Comment:
Since the local `ps` variable seems to be used only here, wouldn't it be better to delete it and use `1.9` here instead?

Reply:
It is more readable to ensure that the expected response is the same variable as the variable usage in method